### ISSUE TITLE
[Bug]: the state of the dropdown "click to customise" is not vocalised on Chips

### ISSUE THREAD

**Issue by afrahB** [Orange-OpenSource/ods-android#279]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### Your test device

Samsung S20

### OS Version

Android 11

### App version

1.0

### Describe the issue

the state of the dropdown "click to customise" is not vocalised

### Expected Behavior

when user (using talkback) navigate on the dropdown "click to customise" on the screen chips, the state should be vocalised (extended or not)
